docs/white_papers.toml

```diff
@@ -15,4 +15,13 @@ pdf_url = "https://github.com/felix-russo/gaussian-track-densities/releases/down
 [white_papers.metadata]
 authors = [ "Felix Russo",]
 title = "Gaussian Track Densities"
-description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the ACTS module AdaptiveGridTrackDensity."
+description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the Acts module AdaptiveGridTrackDensity."
```

Comment:
Is this wanted that the metadata for the previous white papers is updated @paulgessinger?

Reply:
Also, I didn't create a new version since the Gaussian Track Density white paper was created, so this is a bit curious

Reply:
Well, *ACTS* is changed to *Acts* in the abstract.

Reply:
So the change is already in the latest version that you made. So my best guess is the file was not regenerated after you made the release?

Reply:
That is possible!